open-api-spec.yaml

```diff
@@ -850,6 +1030,78 @@ components:
           format: date-time
           pattern: '2022-04-01T10:24:36.11Z'
           description: The date and time when the outgoing payment was updated.
+      required:
+        - id
+        - accountId
```

Comment:
Should `quoteId` be required/exposed?

Reply:
fwiw, Rafiki is using the quote id as the outgoing payment id (to enforce one outgoing payment per quote)
https://github.com/interledger/rafiki/pull/326/files#diff-60554f1175c8f9259e6dc5c51a32a1cf44b6d07301f92fec8f316c63ae87e2ab